src/components/theme/light-theme.css

```diff
@@ -0,0 +1,19 @@
+/* Default and primary button colors differ within modals */
+.light-theme :global(.ant-btn-default) {
+    background-color: var(--light-button-default-background);
+    border-color: var(--light-theme-modal-btn-default-color);
+    color: var(--light-button-default-color);
+}
```

Comment:
Yeah, so our app is mostly dark themed. So these are overrides of the default theme settings to get things like the modals and overlays to have consistent styling instead of having to style each component every time we want a light component. Do you have a suggestion for a better name?

Reply:
"theme" to me suggests a global, swappable color preset - e.g. the way vscode uses "light theme" vs "dark theme." But I can't really think of a better word for this kind of distinction between component styles, so if it's been around in Simularium a while anyways I don't have a problem leaving it.